src/helpers.js

```diff
@@ -162,35 +181,20 @@ function registerWith(Handlebars) {
         }
 
         if (typeof message === 'string') {
-            message = getMessageFormat(message, locales, formats);
+            message = getMessageFormat(isHtml ? message : escape(message), locales, formats);
+        }
+
+        if (hasSafeString) {
+          return new SafeString(String(message.format(hash)));
```

Comment:
`message.format()` will always return a native String, so we can remove the `String()` wrapper around this.